packages/insomnia-inso/src/cli.js

```diff
@@ -171,9 +171,10 @@ export function go(args?: Array<string>, exitOverride?: boolean): void {
   // Global options
   cmd
     .option('-w, --workingDir <dir>', 'set working directory')
-    .option('-a, --appDataDir <dir>', 'set the app data directory')
+    .option('-a, --appDataDir <dir>', 'set the app data directory (deprecated; use --src instead')
     .option('--config <path>', 'path to configuration file')
     .option('--verbose', 'show additional logs while running the command')
+    .option('--src <file|directory>', 'Set the app data source')
```

Comment:
Matching the other options 🤓 

```suggestion
    .option('--src <file|dir>', 'set the app data source')
```